packages/browser-data-collector/package.json

```diff
@@ -17,10 +18,10 @@
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
 	"scripts": {
-		"clean": "tsc --build ./tsconfig.json --clean && tsc --build ./tsconfig-cjs.json --clean",
-		"prepublish": "yarn run clean",
-		"prepare": "tsc --build ./tsconfig.json && tsc --build ./tsconfig-cjs.json",
-		"watch": "tsc --build ./tsconfig.json --watch"
+		"clean": "tsc --build ./tsconfig.json ./tsconfig-cjs.json --clean && npx rimraf dist",
```

Comment:
`tsc --build ... --clean` deletes the files but left lots of empty dirs, that's why I added `npx rimraf dist`